.github/workflows/deploy-gcp-tests.yml

```diff
@@ -1266,6 +1266,18 @@ jobs:
           echo "Found sync height in logs: $SYNC_HEIGHT"
           echo "SYNC_HEIGHT=$SYNC_HEIGHT" >> $GITHUB_ENV
 
+      # Deletes SSH keys generated during this workflow run, as GCP has a limit of SSH keys
+      # that can exist at the same time in the OS Login metadata. Not deleting this keys
+      # could cause the following error:
+      # `Login profile size exceeds 32 KiB. Delete profile values to make additional space`
+      - name: Delete temporal SSH keys
+        continue-on-error: true
+        run: |
+          for i in $(gcloud compute os-login ssh-keys list --format="table[no-heading](value.fingerprint)"); do 
+            echo "$i";
+            gcloud compute os-login ssh-keys remove --key "$i" || true;
+          done
+
```

Comment:
@gustavovalverde should we do this in #5602 and merge?

Reply:
Seems to be needed for every PR to work so I reverted this for now and will wait for https://github.com/ZcashFoundation/zebra/pull/5602 to merge.